eprihlaska/validators.py

```diff
@@ -0,0 +1,27 @@
+import re
+from wtforms import validators
+
+
+class BirthNoValidator(object):
+    def __init__(self, birth_no=-1, message=None):
+        self.birth_no = birth_no
+        if not message:
+            message = 'Zadali ste nesprávne rodné číslo.'
+        self.message = message
+
+    def __call__(self, form, field):
+        bad_form_err = 'Správny formát je RRMMDD/CCCC.'
```

Comment:
@JakubNvk Tieto dve spravy by mohli byt asi parametrizovatelne tak, ako je parametrizovatelna `message`, aby sa to dalo pekne prelozit.